src/main/java/homework3/ex2/page/Ex2HomePage.java

```diff
@@ -0,0 +1,41 @@
+package homework3.ex2.page;
+
+import homework3.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class Ex2HomePage extends BasePage {
+
+    private WebDriver driver;
```

Comment:
you don't need this field here because you have one protected in the parent. Remove



According to Java code convention methods must be lower than fields. Plus should be order:

fields
constructor (if present)
methods

Fix pls